addons/mass_mailing/models/mailing_subscription.py

```diff
@@ -51,3 +51,12 @@ def write(self, vals):
         if vals.get('opt_out_datetime') or vals.get('opt_out_reason_id'):
             vals['opt_out'] = True
         return super().write(vals)
+
+    def open_mailing_contact(self):
+        self.ensure_one()
+        return {
+            'type': 'ir.actions.act_window',
+            'view_mode': 'form',
+            'res_id': self.contact_id.id,
```

Comment:
Could maybe be batch-enabled, and open form view if singleton, tree/form if multi ?

Reply:
Hello @tde-banana-odoo  :slightly_smiling_face: !

I was wondering, how do you know when an action needs to support both singleton and multi? :slightly_smiling_face: 
I see a lot of actions which only supports singleton in the code, and here the action is opened when clicking on a record in the tree view so I don't think it is possible to call it in batch :thinking: 
Is it a preventive measure to support multi, or is there a reason I just don't grasp ? :thinking: 

Thanks! :smile: